boxsdk/session/box_response.py

```diff
@@ -0,0 +1,58 @@
+# coding: utf-8
+
+from __future__ import unicode_literals, absolute_import
+
+
+class BoxResponse(object):
+    """Represents a response to a Box API request."""
+
+    def __init__(self, network_response):
+        self._network_response = network_response
+
+    def json(self):
+        """Return the parsed JSON response.
+
+        :rtype:
+            `dict` or `list` or `str` or `int` or `float`
+        """
+        return self._network_response.json()
+
+    @property
+    def content(self):
+        """Return the content of the response body.
+
+        :rtype:
+            varies
+        """
+        return self._network_response.content
+
+    @property
+    def ok(self):
+        """Return whether or not the request was successful.
+
+        :rtype:
+            `bool`
```

Comment:
Why are these docstrings on 2 lines? vs box_request.py?

Reply:
Because this code is just moved from the `box_session` module, and `box_request` is new code.